app/command_loadtest.go

```diff
@@ -308,6 +327,57 @@ func (me *LoadTestProvider) PostsCommand(a *App, args *model.CommandArgs, messag
 	return &model.CommandResponse{Text: "Added posts", ResponseType: model.COMMAND_RESPONSE_TYPE_EPHEMERAL}
 }
 
+func getMatch(re *regexp.Regexp, text string) string {
+	if match := re.FindStringSubmatch(text); match != nil {
+		return match[1]
+	}
+
+	return ""
+}
+
+func (me *LoadTestProvider) PostCommand(a *App, args *model.CommandArgs, message string) *model.CommandResponse {
+	textMessage := getMatch(messageRE, message)
+	if textMessage == "" {
+		return &model.CommandResponse{Text: "No message to post", ResponseType: model.COMMAND_RESPONSE_TYPE_EPHEMERAL}
+	}
+
+	teamName := getMatch(teamRE, message)
+	team, err := a.GetTeamByName(teamName)
+	if err != nil {
+		return &model.CommandResponse{Text: "Failed to get a team", ResponseType: model.COMMAND_RESPONSE_TYPE_EPHEMERAL}
+	}
+
+	channelName := getMatch(channelRE, message)
+	channel, err := a.GetChannelByName(channelName, team.Id, true)
+	if err != nil {
+		return &model.CommandResponse{Text: "Failed to get a channel", ResponseType: model.COMMAND_RESPONSE_TYPE_EPHEMERAL}
+	}
+
+	passwd := getMatch(passwdRE, message)
+	username := getMatch(userRE, message)
+	user, err := a.GetUserByUsername(username)
+	if err != nil {
+		return &model.CommandResponse{Text: "Failed to get a user", ResponseType: model.COMMAND_RESPONSE_TYPE_EPHEMERAL}
+	}
+
+	client := model.NewAPIv4Client(args.SiteURL)
+	_, resp := client.LoginById(user.Id, passwd)
```

Comment:
Why require a password use the API to create the message? Couldn't we just call `a.CreatePostAsUser` instead?

Reply:
I believed I tried to use `a.CreatePostAsUser` before but it didn't work 'cause it requires UserId.  I'll give it another try.

Reply:
@saturninoabril Are you still trying to make this change, or does it still not work?

Reply:
Thanks for the follow up and sorry for not updating for the past few days as I was working on other stuff.
I gave it another try and the problem with `a.CreatePostAsUser` is on how to get the ID of the user to impersonate.  Besides, though it's for testing only, it might be somehow a protection so that this `/test post` command can only be used in case the user doing the command really knows the password of the other user.

Reply:
My proposal is to keep it this way.  Please let me know if you agree with it, thanks

Reply:
That's a good point that it adds that extra protection. I think it's still a bit weird to do it this way, but it's fine since it's just to help testing.